test/views/declaration/summary/ReferencesSectionViewSpec.scala

```diff
@@ -75,15 +76,32 @@ class ReferencesSectionViewSpec extends UnitViewSpec with ExportsTestData with I
       row must haveSummaryActionsHref(routes.ConsignmentReferencesController.displayPage(Mode.Change))
     }
 
-    "have LRN with change button" in {
+    onJourney(STANDARD, SIMPLIFIED, OCCASIONAL, CLEARANCE) { implicit request =>
+      "have LRN with change button" in {
 
-      val row = view.getElementsByClass("lrn-row")
-      row must haveSummaryKey(messages("declaration.summary.references.lrn"))
-      row must haveSummaryValue("LRN")
+        val row = view.getElementsByClass("lrn-row")
+        row must haveSummaryKey(messages("declaration.summary.references.lrn"))
+        row must haveSummaryValue("LRN")
 
-      row must haveSummaryActionsTexts("site.change", "declaration.summary.references.lrn.change")
+        row must haveSummaryActionsTexts("site.change", "declaration.summary.references.lrn.change")
 
-      row must haveSummaryActionsHref(routes.ConsignmentReferencesController.displayPage(Mode.Change))
+        row must haveSummaryActionsHref(routes.ConsignmentReferencesController.displayPage(Mode.Change))
+      }
+    }
```

Comment:
Actually, in this test case you don't have different journeys, because you are using the `view` value. Which is defined once with Standard journey data